csharp/lib/AsyncClient.cs

```diff
@@ -127,107 +127,528 @@ private void SuccessCallback(ulong index, IntPtr str)
+        PfAdd = 701,
+        PfCount = 702,
+        PfMerge = 703,
+
+        //// List commands: https://redis.io/docs/latest/commands/?group=list
+        BLMove = 801,
+        BLMPop = 802,
+        BLPop = 803,
+        BRPop = 804,
+        BRPopLPush = 805, // deprecated in 6.2.0
+        LIndex = 806,
+        LInsert = 807,
+        LLen = 808,
+        LMove = 809,
+        LMPop = 810,
+        LPop = 811,
+        LPos = 812,
+        LPush = 813,
+        LPushX = 814,
+        LRange = 815,
+        LRem = 816,
+        LSet = 817,
+        LTrim = 818,
+        RPop = 819,
+        RPopLPush = 820, // deprecated in 6.2.0
+        RPush = 821,
+        RPushX = 822,
+
+        //// Pub/Sub commands: https://redis.io/docs/latest/commands/?group=pubsub
+        PSubscribe = 901,
+        Publish = 902,
+        PubSubChannels = 903,
+        PubSubNumPat = 904,
+        PubSubNumSub = 905,
+        PubSubShardChannels = 906,
+        PubSubShardNumSub = 907,
+        PUnsubscribe = 908,
+        SPublish = 909,
+        SSubscribe = 910,
+        Subscribe = 911,
+        SUnsubscribe = 912,
+        Unsubscribe = 913,
+
+        //// Scripting and Functions commands: https://redis.io/docs/latest/commands/?group=scripting
+        Eval = 1001,
+        EvalRo = 1002,
+        EvalSha = 1003,
+        EvalShaRo = 1004,
+        FCall = 1005,
+        FCallRo = 1006,
+        FunctionDelete = 1007,
+        FunctionDump = 1008,
+        FunctionFlush = 1009,
+        FunctionKill = 1010,
+        FunctionList = 1011,
+        FunctionLoad = 1012,
+        FunctionRestore = 1013,
+        FunctionStats = 1014,
+        ScriptDebug = 1015,
+        ScriptExists = 1016,
+        ScriptFlush = 1017,
+        ScriptKill = 1018,
+        ScriptLoad = 1019,
+
+        //// Server management commands: https://redis.io/docs/latest/commands/?group=server
+        AclCat = 1101,
+        AclDelUser = 1102,
+        AclDryRun = 1103,
+        AclGenPass = 1104,
+        AclGetUser = 1105,
+        AclList = 1106,
+        AclLoad = 1107,
+        AclLog = 1108,
+        AclSave = 1109,
+        AclSetSser = 1110,
+        AclUsers = 1111,
+        AclWhoami = 1112,
+        BgRewriteAof = 1113,
+        BgSave = 1114,
+        Command = 1115,
+        CommandCount = 1116,
+        CommandDocs = 1117,
+        CommandGetKeys = 1118,
+        CommandGetKeysAndFlags = 1119,
+        CommandInfo = 1120,
+        CommandList = 1121,
+        ConfigGet = 1122,
+        ConfigResetStat = 1123,
+        ConfigRewrite = 1124,
+        ConfigSet = 1125,
+        DbSize = 1126,
+        FailOver = 1127,
+        FlushAll = 1128,
+        FlushDb = 1129,
+        Info = 1130,
+        LastSave = 1131,
+        LatencyDoctor = 1132,
+        LatencyGraph = 1133,
+        LatencyHistogram = 1134,
+        LatencyHistory = 1135,
+        LatencyLatest = 1136,
+        LatencyReset = 1137,
+        Lolwut = 1138,
```

Comment:
I can't believe this is actually real... https://redis.io/docs/latest/commands/lolwut/

Reply:
I tested it with custom command and looking forward adding implementation of that!